index.html

```diff
@@ -61,6 +61,165 @@ <h1 class="intro primary-text1">
+                    <li class="languages">html</li>
+                    <li class="languages">css</li>
+                    <li class="languages">javascript</li>
+                </ul>
+                <button type="button" class="btn-enabled">
+                    See Project
+                </button>
+            </div>
+            <div class="card">
+                <img class="profile-image" src="images/image_3.png" alt="">
+                <h2>Tonic</h2>
+                <ul class="icons">
+                    <li class="canopy">CANOPY</li>
+                    <li class="counter"><img src="images/Counter.svg" alt=""></li>
+                    <li class="bg">Back End Dev</li>
+                    <li class="counter"><img src="images/Counter.svg" alt=""></li>
+                    <li class="bg">2015</li>
+                </ul>
+                <p class="profile-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="icons">
+                    <li class="languages">html</li>
+                    <li class="languages">css</li>
+                    <li class="languages">javascript</li>
+                </ul>
+                <button type="button" class="btn-enabled">
+                    See Project
+                </button>
+            </div>
+            <div class="card">
+                <img class="profile-image" src="images/image_4.png" alt="">
+                <h2>Multi-Post Stories</h2>
+                <ul class="icons">
+                    <li class="canopy">CANOPY</li>
+                    <li class="counter"><img src="images/Counter.svg" alt=""></li>
+                    <li class="bg">Back End Dev</li>
+                    <li class="counter"><img src="images/Counter.svg" alt=""></li>
+                    <li class="bg">2015</li>
+                </ul>
+                <p class="profile-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="icons">
+                    <li class="languages">html</li>
+                    <li class="languages">css</li>
+                    <li class="languages">javascript</li>
+                </ul>
+                <button type="button" class="btn-enabled">
+                    See Project
+                </button>
+            </div>
+        </section>
+        <section id="about">
+            <h1 class="about">About Myself</h1>
+            <p class="about-content">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="#" class="connect">
+                LET'S CONNECT
+            </a>
```

Comment:
Please use `hx` `(e.g h1, h2, h3)` as requested by the previous reviewer. Because this is a heading not a link to a page and this is part of the requirement.